accounts/views.py

```
@@ -2,2 +1,31 @@
+@method_decorator(require_GET, name='dispatch')
+class ProfileView(LoginRequiredMixin, TemplateView):
+    template_name = 'profile.html'
+
+
+class ProfileUpdateView(LoginRequiredMixin, UpdateView):
+    model = Profile
+    template_name = 'edit_profile.html'
+    form_class = ProfileUpdateForm
+    success_url = reverse_lazy('profile')
+
+    def form_valid(self, form):
```

Ruff (0.8.2)

Missing return type annotation for public function `form_valid`

(ANN201)